components/wpt-results.html

```diff
@@ -138,12 +138,20 @@
           testRuns: {
             type: Array,
           },
+          filteredDirectories: {
+            type: Array,
+            value: []
+          },
         };
```

Comment:
Yes definitely, the bug for linting is #11, I hope I can get more time this week to work on WPTD and set up things like that.